src/store/map/index.js

```diff
@@ -1,34 +1,19 @@
-import { mapBoxClient, mayaClient } from '@/services/api';
-import _ from 'lodash';
+import { mayaClient } from '@/services/api';
+import { firebase, getDatabase, ref, get, child } from '../../firebase';
 
 const state = {
     locations: [],
-    selectedLocation: {
-        city: '',
-        state: '',
-        country: '',
-        locName: '',
-    },
-    mapConfig: {
-        container: 'map',
-        style: 'mapbox://styles/mapbox/streets-v11',
-        center: [77.5946, 12.9716], //  default bengaluru lat, lng.
-        zoom: 11,
-    },
-    center: null,
-    totalPages: 1, // default page number
+    selectedLocation: null,
+    selectedLocationLatLng: null,
+    mapCenter: { lat: 17.471356, lng: 78.3344256 }, //  default bengaluru lat, lng.
     srpResults: [],
-    paginateSrpResults: [],
     recentSearch: [],
     recentID: 0,
+    GOOGLE_TOKEN: '',
     filteredSrpResults: [],
 };
 
 const getters = {
-    getMapConfig(state) {
-        return state.mapConfig;
-    },
-
     getLocationName(state) {
```

Comment:
we can avoid simple getters
"Sometimes we may need to compute derived state based on store state, for example filtering through a list of items and counting them:"
https://vuex.vuejs.org/guide/getters.html